packages/core/src/transport/eventBridge.ts

```diff
@@ -1,25 +1,35 @@
-import { isExperimentalFeatureEnabled } from '..'
+import { includes, isExperimentalFeatureEnabled } from '..'
 
 export interface BrowserWindow extends Window {
   DatadogEventBridge?: DatadogEventBridge
 }
 
 export interface DatadogEventBridge {
+  getAllowedWebViewHosts(): string
   send(msg: string): void
 }
 
 export function getEventBridge<T, E>() {
-  const datadogEventBridge = getEventBridgeGlobal()
+  const eventBridgeGlobal = getEventBridgeGlobal()
+
+  if (!eventBridgeGlobal) {
+    return { getAllowedWebViewHosts: () => [], send: () => undefined }
+  }
 
   return {
+    getAllowedWebViewHosts() {
+      return JSON.parse(eventBridgeGlobal.getAllowedWebViewHosts()) as string[]
+    },
     send(eventType: T, event: E) {
-      datadogEventBridge?.send(JSON.stringify({ eventType, event }))
+      eventBridgeGlobal.send(JSON.stringify({ eventType, event }))
     },
   }
 }
 
 export function isEventBridgePresent(): boolean {
-  return !!getEventBridgeGlobal()
+  return getEventBridge()
+    .getAllowedWebViewHosts()
+    .some((o) => includes(o, window.location.hostname))
```

Comment:
```suggestion
    .some((host) => includes(host, window.location.hostname))
```

Reply:
why `includes` and not `===`? 
should we add a test for it?

Reply:
It was to handle port but know that I see it, we might have some issue with the include. I've double checked with Marius and an equality will do.

Reply:
For the test it's done within the `isEventBridgePresent`.